test/Aqua.jl

```diff
@@ -0,0 +1,15 @@
+using Aqua
+
+@testset "Aqua.jl" begin
+    Aqua.test_all(
+        GAP;
+        ambiguities=false,      # TODO: fix ambiguities
+        unbound_args=true,
+        undefined_exports=true,
+        project_extras=true,
+        stale_deps=false,       # some weird error with GAP_lib_jll
+        deps_compat=true,
+        project_toml_formatting=true,
+        piracy=false            # TODO: fix
```

Comment:
Note that I believe the reported issues are not actuality type piracy but rather a bug in Aqua; I already reported this there, too

Reply:
I haven't looked into everything that has been reported by Aqua and just disabled the failing checks for the time being. This should be tackled in future PRs.